src/ext/java/org/opentripplanner/ext/transmodelapi/support/GqlUtil.java

```diff
@@ -104,4 +106,14 @@ public static boolean hasArgument(DataFetchingEnvironment environment, String na
   public static <T> List<T> listOfNullSafe(T element) {
     return element == null ? List.of() : List.of(element);
   }
+
+  /**
+   * Helper method to support the deprecated 'lang' argument.
+   */
+  public static Locale getLocale(DataFetchingEnvironment environment) {
```

Comment:
I'm not going to insist on it, since it's your sandbox feature, but if you want you can write a test for this. Check out `GraphQLUtilsTest` for an example of how to construct a `DataFetchingEnvironemt`.